src/blog/2023/10/dashboard-integrations.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Integrate your own widgets with Dashboard 2.0
+subtitle: With the 0.6.0 Release of Dashboard 2.0, we now support third-party widget integration. Read more in this deep dive.
+description: With the 0.6.0 Release of Dashboard 2.0, we now support third-party widget integration. Read more in this deep dive.
+date: 2023-10-06
+authors: ["joe-pavitt"]
+image: "/blog/2023/10/images/tile-db-integration.png"
+tags:
+    - posts
+    - releases
+    - community
+    - changelog
+---
+
+With a new release, comes new features for Dashboard 2.0, and the focus of this release has been on improving the developer experience for those building third-party widgets for Dashboard 2.0.
+
+<!--more-->
+
+Dashboard 1.0 had a hugely popular ecosystem of third party widgets (e.g. `ui-worldmap`, `ui-svg`) and something we've been keen to support is a platform where these widgets (and more) can be built and used within Dashboard 2.0 too.
+
+Whilst we can't support the existing Dashboard 1.0 extensions directly (given that we're now VueJS-based, rather than AngularJS), we hope that the framework, documentation and this article, will help springboard the community to build new (and transfer over old) widgets for Dashboard 2.0.
```

Comment:
Whilst vs While, not sure if you're correct?

Reply:
"While" suggests that things will change, "Whilst" is more appropriate here